internal/model/model.go

```diff
@@ -42,6 +101,19 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				m.Page--
 			}
 		}
+
+	case fileWatchMsg:
+		newFileInfo, err := os.Stat(m.FileName)
+		if err == nil {
+			if newFileInfo.ModTime() != fileInfo.ModTime() {
```

Comment:
We are pretty much implementing our own polling method every one second, is there any way to use something like  https://github.com/fsnotify/fsnotify so we don't have to poll and we get instant updates.

Reply:
I'm not so familiar with bubbletea library to simulate `tea.Msg` event and integrate seamlessly it into the render loop. On the other hand, in the case of `slides` app one stats call per second is not so expensive. Switch to inotify/fsevent seems like will give almost zero performance benefit.

Reply:
Sounds good, its mainly to have instant reloading rather than 1 second delayed reloading.

Reply:
I might have a look later on how to use fsnotify